scripts/ingests/ingest_utils.py

```diff
@@ -22,7 +22,7 @@
 
 # SOURCES
 def ingest_sources(db, sources, references=None, ras=None, decs=None, comments=None, epochs=None,
-                   equinoxes=None, raise_error=True, search_db=True):
+                   equinoxes=None, other_references=None, raise_error=True, search_db=True):
```

Comment:
May want to update the docstring since `other_references` is new and not defined there